src/views/ApplicationPage/majors.js

```diff
@@ -0,0 +1,96 @@
+const MajorsList = [

```

Comment:
move this file to the config folder (either use the majors file that's there already or replace it with this)